packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -269,6 +269,12 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('pausePreview() is not implemented.');
   }
 
+  /// Set the active camera while recording.
```

Comment:
Nit: Sets

(Several of the comments above are wrong, but not all, so new comments should use the standard style.)